hetu-docs/en/admin/properties.md

```diff
@@ -277,6 +277,23 @@ This section describes the most important config properties that may be used to
 >
 > Enables to prioritize splits with larger revocable memory.
 
+### `experimental.spiller-spill-to-hdfs`
+
+> -   **Type:** `boolean`
+> -   **Default value:** `flase`
+>
+> Enables spilling into HDFS. When this property is set to `true` the property `experimental.spiller-spill-profile` must be set and also `experimental.spiller-spill-path` must contain only a single path.
+
+### `experimental.spiller-spill-profile`
+
+> -   **Type:** `string`
+> -   **No default value.** Must be set when spilling to hdfs is enabled
+>
+>
+> This property defines the [filesystem](../develop/filesystem.md) profile used to spill. The corresponding profile must exist in `etc/filesystem`. For example, if this property is set as `experimental.spiller-spill-profile=spill-hdfs`, a profile describing this filesystem `spill-hdfs.properties` must be created in `etc/filesystem` with necessary information including authentication type, config, and keytabs (if applicable). Please refer to the [filesystem](../develop/filesystem.md) section for details.
```

Comment:
no need to "Please refer", just give reference is enough

Reply:
Resolved